docs/Pflichtenheft.tex

```diff
@@ -199,124 +200,118 @@ \subsection{Hardware}
-  \item[FA420] Die \gls{Uberwachungskonsole} zeigt die Zuflussmenge der oberen Tanks auf jeweils einer Skala an.
-  \item[FA430] Die \gls{Uberwachungskonsole} zeigt die Abflussmenge aller Tanks auf jeweils einer Skala an.
-  \item[FA440] Die \gls{Uberwachungskonsole} zeigt den Füllstand aller Tanks auf jeweils einer Skala an.
-  \item[FA450] Die \gls{Uberwachungskonsole} zeigt die Temperatur aller Tanks auf jeweils einer Skala an (sofern implementiert).
-  \item[FA460] Die \gls{Uberwachungskonsole} zeigt die Drehzahl des Mischermotors im unteren Tank auf einer Skala an.
-  \item[FA470] Der zeitliche Verlauf des Füllstandes wird jeweils für alle Tanks angezeigt.
-  \item[FA480] Der zeitliche Verlauf der Temperatur wird jeweils für alle Tanks angezeigt.
-  \item[FA490] Die \gls{Uberwachungskonsole} registriert für jeden Tank jeweils einen Alarm für einen Überlauf.
-  \item[FA500] Die \gls{Uberwachungskonsole} registriert für jeden Tank jeweils einen Alarm für einen Unterlauf.
-  \item[FA510] Ein Alarm wird durch einen Bezeichner und einen Kreis als Zustandssymbol dargestellt.
-  \item[FA520] Ist der Kreis rot, wurde der Alarm ausgelöst.
-  \item[FA530] Ist der Kreis grün, ist der Alarm nicht ausgelöst.
-  \item[FA540] Ein Alarm wird als ausgelöst angezeigt, nachdem die \gls{Uberwachungskonsole} durch die \gls{Fertigungssimulation} darüber benachrichtigt wurde und bis der Alarm in der
+  \item[FA450] Die \gls{Uberwachungskonsole} stellt die \glspl{Sensordatum} und Alarme jedes Flüssigkeitsanks in einem eigenen Bereich dar.
+  \item[FA460] Die \gls{Uberwachungskonsole} zeigt die Farbe der Tanks an.
+  \item[FA470] Die \gls{Uberwachungskonsole} zeigt die Zuflussmenge der oberen Tanks auf jeweils einer Skala an.
+  \item[FA480] Die \gls{Uberwachungskonsole} zeigt die Abflussmenge aller Tanks auf jeweils einer Skala an.
+  \item[FA490] Die \gls{Uberwachungskonsole} zeigt den Füllstand aller Tanks auf jeweils einer Skala an.
+  \item[\textcolor{blue}{FA500}] Die \gls{Uberwachungskonsole} zeigt die Temperatur aller Tanks auf jeweils einer Skala an.
+  \item[FA510] Die \gls{Uberwachungskonsole} zeigt die Drehzahl des Mischermotors im unteren Tank auf einer Skala an.
+  \item[FA520] Der zeitliche Verlauf des Füllstandes wird jeweils für alle Tanks angezeigt.
+  \item[FA530] Der zeitliche Verlauf der Temperatur wird jeweils für alle Tanks angezeigt.
+  \item[FA540] Die \gls{Uberwachungskonsole} registriert für jeden Tank jeweils einen Alarm für einen Überlauf.
+  \item[FA550] Die \gls{Uberwachungskonsole} registriert für jeden Tank jeweils einen Alarm für einen Unterlauf.
+  \item[FA560] Ein Alarm wird durch einen Bezeichner und einen Kreis als Zustandssymbol dargestellt.
+  \item[FA570] Ist der Kreis rot, wurde der Alarm ausgelöst.
+  \item[FA580] Ist der Kreis grün, ist der Alarm nicht ausgelöst.
+  \item[\textcolor{blue}{FA590}] Der Benutzer kann die Schwellenwerte der Alarme direkt an deren Anzeige ändern.
```

Comment:
Mir ist nicht so ganz klar, was damit gemeint ist. Heißt an der Anzeige, wenn der Alarm ausgelöst ist? Oder bei den Checkboxen in den Einstellungen?